File: api-tester/src/main/java/com/revenuecat/apitester/java/PurchaseOptionAPI.java
package com.revenuecat.apitester.java;

import com.revenuecat.purchases.google.GooglePurchaseOption;
import com.revenuecat.purchases.models.PricingPhase;
import com.revenuecat.purchases.models.PurchaseOption;

import java.util.List;

@SuppressWarnings({"unused"})
final class PurchaseOptionAPI {

    static void checkPurchaseOption(PurchaseOption purchaseOption) {
        List<PricingPhase> pricingPhases = purchaseOption.getPricingPhases();
        List<String> tags = purchaseOption.getTags();
        Boolean isBasePlan = purchaseOption.isBasePlan();
    }

}

File: api-tester/src/main/java/com/revenuecat/apitester/kotlin/PurchaseOptionApi.kt
package com.revenuecat.apitester.kotlin

import com.revenuecat.purchases.models.PricingPhase
import com.revenuecat.purchases.models.PurchaseOption

@Suppress("unused", "UNUSED_VARIABLE", "RemoveExplicitTypeArguments")
private class PurchaseOptionApi {

    fun checkPurchaseOption(purchaseOption: PurchaseOption) {
        val phases: List<PricingPhase> = purchaseOption.pricingPhases
        val tags: List<String> = purchaseOption.tags
        val isBasePlan: Boolean = purchaseOption.isBasePlan
    }
}
